js/switcher.js

```diff
@@ -80,6 +80,25 @@ const SwitcherSingleResultView = (r, isSelected, selectResult) => {
 
 class Switcher extends Component {
   handleKeyDown(e) {
+    if (e.target === this.inputNode) {
```

Comment:
Moved arrow up/down handling here, so it's a little bit better experience – now selection will move when on hold instead of single-step per keystroke.